feature-libs/customer-ticketing/components/customer-ticketing-create/customer-ticketing-create-dialog/customer-ticketing-create-dialog.component.ts

```diff
@@ -1,23 +1,83 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, Input, OnDestroy, OnInit } from '@angular/core';
+import {
+  AssociatedObject,
+  Category,
+  Ticket,
+} from '@spartacus/customer-ticketing/root';
 import { FormUtils } from '@spartacus/storefront';
+import { Observable, Subscription } from 'rxjs';
 import { CustomerTicketingDialogComponent } from '../../shared/customer-ticketing-dialog/customer-ticketing-dialog.component';
-
 @Component({
   selector: 'cx-customer-ticketing-create-dialog',
   templateUrl: './customer-ticketing-create-dialog.component.html',
 })
 export class CustomerTicketingCreateDialogComponent
   extends CustomerTicketingDialogComponent
-  implements OnInit
+  implements OnInit, OnDestroy
 {
+  ticketCategories$: Observable<Category[]> =
+    this.customerTicketingFacade.getTicketCategories();
+  ticketAssociatedObjects$: Observable<AssociatedObject[]> =
+    this.customerTicketingFacade.getTicketAssociatedObjects();
+  subscription: Subscription;
+
+  @Input()
+  selectedCategory: Category;
+
+  @Input()
+  selectedAssociatedTo: AssociatedObject;
```

Comment:
```suggestion
  selectedAssociatedObject: AssociatedObject;
```
Line 79 in the correspondent template lines above uses "selectedAssociatedObject"
I know the payload requires "associatedTo", but it is consistent to selected associated object